network-area-diagram/src/main/java/com/powsybl/nad/svg/iidm/DefaultLabelProvider.java

```diff
@@ -94,7 +93,30 @@ public String getBusDescription(BusNode busNode) {
 
     @Override
     public List<String> getVoltageLevelDetails(VoltageLevelNode vlNode) {
-        return Collections.emptyList();
+        List<String> voltageLevelDetails = new ArrayList<>();
+        VoltageLevel voltageLevel = network.getVoltageLevel(vlNode.getEquipmentId());
+
+        double activeProductionValue = voltageLevel.getGeneratorStream().mapToDouble(generator -> !Double.isNaN(-generator.getTerminal().getP()) ? -generator.getTerminal().getP() : 0).sum();
```

Comment:
This stream operation (and the 3 others) can be simplified that way. Besides it will avoid the complexity issue raised by sonar.
```suggestion
        double activeProductionValue = voltageLevel.getGeneratorStream().mapToDouble(generator -> -generator.getTerminal().getP()).filter(p -> !Double.isNaN(p)).sum();
```